### PR TITLE
[TASK-15] add jest for testing

### PR DESCRIPTION
**Pull request by nugrazurus**
- Add modules:
  - jest
  - ts-jest
  - supertest
- add app.test.ts

### CODE REVIEW

**Comment by nugrazurus** [comment]
@fikrialwan Tolong review pak

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
   "scripts": {
     "build": "tsc",
     "start": "nodemon dist/server.js",
-    "dev": "nodemon src/server.ts"
+    "dev": "nodemon src/server.ts",
+    "test": "jest",
+    "test:watch": "jest --watchAll"
```
Comment (fikrialwan) [inline package.json]: I think you can add script for coverage
`"test:coverage": "jest --coverage"`

**Comment by fikrialwan** [comment]
Hai @nugrazurus I have one feedback for adding script for runing test coverage

**Review by fikrialwan — COMMENTED** [review]
oke cool

**Pull request merged**